tests/lib/Controller/DiscoPowerTest.php

```diff
@@ -9,7 +9,7 @@
 use SimpleSAML\Error;
 use SimpleSAML\Module\discopower\Controller;
 use SimpleSAML\Session;
-use SimpleSAML\Test\Utils\ClearStateTestCase;
+use SimpleSAML\TestUtils\ClearStateTestCase;
```

Comment:
Thing is, it works for me locally when I use SSP's SimpleSAML\Test\Utils\ClearStateTestCase, but does not work when using SimpleSAML\TestUtils\ClearStateTestCase....

Reply:
Ah yes I spotted a bug in the test-framework... v1.1.1 coming up!

Reply:
There ya go, all tests pass now